src/contour/ui.template/Terminal.qml.in

```diff
@@ -28,6 +28,31 @@ ContourTerminal
         focus : false
     }
 
+
+
+    Rectangle {
+        Timer {
+            id: sizeWidgetTimer
+            interval: 1000;
+            running: false;
+            onTriggered: sizeWidget.visible = false
+        }
+        id : sizeWidget
+        anchors.centerIn: parent
+        width:  80
```

Comment:
can we somehow set this to auto? to have it auto-grow on larger text? (should rarely happen, but still)

Reply:
we can not set it to auto but i added some logic so we have some minimal value and then scale to 10% of width and height